modules/ossm-config-openshift-monitoring-only.adoc

```diff
@@ -87,10 +94,17 @@ spec:
       targetLabel: pod_name
 ----
 <1> Because {ocp-product-title} monitoring ignores the `namespaceSelector` spec in `ServiceMonitor` and `PodMonitor` objects, you must apply the `PodMonitor` object in all mesh namespaces, including the Istio control plane namespace.
+
+. Apply the YAML file by running the following command:
 +
-. Go to the {ocp-short-name} Console -> **Observe** -> **Metrics**, and run the query `istio_requests_total`.
+[source,terminal]
+----
+$ oc apply -f podmonitor.yml
+----
 
+. Go to the {ocp-short-name} Console -> **Observe** -> **Metrics**, and run the query `istio_requests_total`.
```

Comment:
I understand that this wording existed before this PR, but I feel that it sounds a little awkward to include the console on the "chain" of pages/options to follow here. Usually I think I see things like "On the console, go to Observe->Metrics". 

Not something I will stop a merge over, but would you consider rewording to something more like this?
```suggestion
. On the {ocp-short-name} Console, go to **Observe** -> **Metrics** and run the query `istio_requests_total`.
```

Reply:
done